vmdb/lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -38,6 +38,8 @@ class MiqAeObject
       'server'                 => 'MiqServer'
     )
     attr_accessor :attributes, :namespace, :klass, :instance, :object_name, :instance_methods, :workspace, :current_field, :current_message
+    attr_accessor :parent
```

Comment:
We could change it to `node_parent`?  I changed it to `node_children` for the children since we already have a method named `children`.